core/router/gateway/monitor.go

```diff
@@ -1,102 +1,191 @@
 package gateway
 
 import (
```

Comment:
Please clean up imports. There should not be an empty line after `google.golang.org/grpc/metadata` nor after `context "golang.org/x/net/context`.